CHANGES/898.feature.rst

```diff
@@ -0,0 +1 @@
+Added ``without_query_params`` method -- by :user:`hongquan`.
```

Comment:
Could you make this more detailed? As in, mention where the method is added to, and so on. With end-users being the audience in mind.